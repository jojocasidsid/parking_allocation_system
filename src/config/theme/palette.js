const blue = '#002F5D';
const black = '#282B2F';
const white = '#FFFFFF';

const exportColors = {
  colors: {
    blue,
    black,
    white
  },
  primary: {
    main: blue
  },
  secondary: {
    main: black
  },
  info: {
    main: white
  },
  text: {
    primary: black,
    secondary: '#B5B5B5'
  },
  action: {
    disabled: black
  }
};

export default exportColors;
