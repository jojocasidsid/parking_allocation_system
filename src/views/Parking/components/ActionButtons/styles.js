import styled from '@emotion/styled';

export const ActionWrapper = styled('div')`
  display: flex;
  justify-content: space-around;
  align-items: center;
  padding: 20px;
`;
