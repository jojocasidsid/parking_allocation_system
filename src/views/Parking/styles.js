import styled from '@emotion/styled';

export const SlotsWrapper = styled('div')`
  display: grid;
  grid-template-columns: repeat(5, 1fr);
  grid-template-rows: repeat(5, 1fr);
  grid-gap: 10px;
`;

export const ActionWrapper = styled('div')`
  display: flex;
  justify-content: space-around;
  align-items: center;
  padding: 20px;
`;
