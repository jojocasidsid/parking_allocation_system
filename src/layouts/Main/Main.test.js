import React from 'react';
import renderer from 'react-test-renderer';
import { ThemeProvider } from '@mui/material/styles';

import theme from 'config/theme/theme';

import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';

import MainLayout from '.';

describe('Main layout', () => {
  it('should render without crashing', () => {
    const tree = renderer.create(
      <ThemeProvider theme={theme}>
        <Router>
          <Routes>
            <Route path="*" element={<MainLayout />} />
          </Routes>
        </Router>
      </ThemeProvider>
    );

    expect(tree).toMatchSnapshot();
  });
});
