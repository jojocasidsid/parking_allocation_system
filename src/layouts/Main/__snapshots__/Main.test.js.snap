// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main layout should render without crashing 1`] = `
.emotion-0 {
  background-color: #fff;
  color: #282B2F;
  -webkit-transition: box-shadow 300ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: box-shadow 300ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  box-shadow: 0px 2px 4px -1px rgba(0,0,0,0.2),0px 4px 5px 0px rgba(0,0,0,0.14),0px 1px 10px 0px rgba(0,0,0,0.12);
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  width: 100%;
  box-sizing: border-box;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  position: static;
  background-color: #002F5D;
  color: #fff;
  margin-bottom: 20px;
}

.emotion-1 {
  position: relative;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  min-height: 56px;
}

@media (min-width:0px) and (orientation: landscape) {
  .emotion-1 {
    min-height: 48px;
  }
}

@media (min-width:600px) {
  .emotion-1 {
    min-height: 64px;
  }
}

.emotion-2 {
  margin-left: 20px;
  width: 50px;
}

.emotion-3 {
  margin-left: 16px;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
}

@media (min-width:0px) {
  .emotion-3 {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
  }
}

@media (min-width:900px) {
  .emotion-3 {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
  }
}

.emotion-4 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  position: relative;
  box-sizing: border-box;
  -webkit-tap-highlight-color: transparent;
  background-color: transparent;
  outline: 0;
  border: 0;
  margin: 0;
  border-radius: 0;
  padding: 0;
  cursor: pointer;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  vertical-align: middle;
  -moz-appearance: none;
  -webkit-appearance: none;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: inherit;
  color: #FFFFFF;
  font-style: normal;
  font-weight: 600;
  font-size: 16x;
  line-height: 22px;
  letter-spacing: -0.02em;
  font-family: Rubik;
  text-transform: uppercase;
  min-width: 64px;
  padding: 6px 8px;
  border-radius: 4px;
  -webkit-transition: background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
}

.emotion-4::-moz-focus-inner {
  border-style: none;
}

.emotion-4.Mui-disabled {
  pointer-events: none;
  cursor: default;
}

@media print {
  .emotion-4 {
    -webkit-print-color-adjust: exact;
    color-adjust: exact;
  }
}

.emotion-4:hover {
  -webkit-text-decoration: none;
  text-decoration: none;
  background-color: rgba(255, 255, 255, 0.04);
}

@media (hover: none) {
  .emotion-4:hover {
    background-color: transparent;
  }
}

.emotion-4.Mui-disabled {
  color: #282B2F;
}

.emotion-5 {
  -webkit-box-flex: 0;
  -webkit-flex-grow: 0;
  -ms-flex-positive: 0;
  flex-grow: 0;
  margin-right: 16px;
}

.emotion-6 {
  -webkit-box-flex: 0;
  -webkit-flex-grow: 0;
  -ms-flex-positive: 0;
  flex-grow: 0;
}

.emotion-7 {
  margin: 20px;
}

<div
  className="main-root"
>
  <header
    className="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionStatic emotion-0"
  >
    <div
      className="main-container"
    >
      <div
        className="MuiToolbar-root MuiToolbar-regular emotion-1"
      >
        <img
          alt="logo"
          className="image emotion-2"
          src="https://deltek.com/favicon.ico"
        />
        <div
          className="MuiBox-root emotion-3"
        >
          <a
            className="MuiButton-root MuiButton-text MuiButton-textInfo MuiButton-sizeMedium MuiButton-textSizeMedium MuiButtonBase-root  emotion-4"
            href="/contacts"
            onBlur={[Function]}
            onClick={[Function]}
            onContextMenu={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={0}
          >
            Contacts
          </a>
        </div>
        <div
          className="MuiBox-root emotion-5"
        />
        <div
          className="MuiBox-root emotion-6"
        />
      </div>
    </div>
  </header>
  <div
    className="emotion-7"
  />
</div>
`;
